src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/extension/env/EnvExtension.groovy

```diff
@@ -73,6 +73,9 @@ class EnvExtension extends AbstractGlobalExtension implements SpringContextListe
     @Value('${discovery.timeout}')
     int discoveryTimeout
 
+    @Value('${reset.lab_state}')
```

Comment:
Let's make sure that we behave as before this change by default. Add default value 'true' here:
```
@Value('${reset.lab_state:true}')